mindmeld/cli.py

```diff
@@ -531,6 +535,32 @@ def _get_duckling_pid():
     return pid
 
 
+@shared_cli.command("annotate", context_settings=CONTEXT_SETTINGS)
+@click.option(
+    "--app-path", required=True, help="Needed to locate the application config file.",
+)
+def annotate(app_path):
+    """Runs the annotation command of the Auto Annotator."""
+    importlib.import_module("mindmeld.auto_annotator")
```

Comment:
why is this for?

Reply:
The auto_annotator needs to be loaded dynamically here because at bottom of the auto_annotator.py file the SpacyAnnotator is registered. This register function comes from models/helpers.py.

auto_annotator.py (ln 708): register_annotator("SpacyAnnotator", SpacyAnnotator)

Importing the module at the top of the file creates a pylint error since the module isn't directly used but just loaded so that ANNOTATOR_MAP = {} in models/helpers.py can be updated

Reply:
You can just do a pylint ignore for the module import at the top; we would like to avoid dynamic import as much as possible

Reply:
Got it, done